packages/clay-css/src/scss/variables/_buttons.scss

```diff
@@ -892,6 +892,19 @@ $btn-palette: map-deep-merge(
 		light: $btn-light,
 		dark: $btn-dark,
 		link: $btn-link,
+		btn-beta: (
```

Comment:
Why are we prefixing beta with btn, instead of letting btn be added automatically like it is for the other variants?

Reply:
Same reason as https://github.com/liferay/clay/pull/5447#discussion_r1154767684, I wanted to convert these to have the `btn-` prefix, but can't due to backward compatibility.